docs/2023/Farmstack/Dashboard Builder/updates/2023-07-28.md

```diff
@@ -4,13 +4,30 @@ author: Jhansi
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- I have created a sidebar which when opened displays the created measures if created any
```

Comment:
Add checkbox for signifying milestone completion

Reply:
Actually I have been mentioning the milestones that I have achieved.But from next pull request I will follow this and also edit the previous ones as you told.